ProcessMaker/modeler
# FOUR-16765: The API calls in modeler is execute twice

## Issue & Reproduction Steps

1. Go to modeler 
2. Create a process and a task with Screens and Display Next Assigned Task to Task Assignee enable
3. Open the debuger
4. Click on any task and see th calls in Network

## Solution
- Add validation to Inspector Panel in Modeler

## How to Test
- Go to modeler 
- Create a process and a task with Screens and Display Next Assigned Task to Task Assignee enable
- Open the browser debugger
- Click on any task and see the calls in Network

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-16765

## Code Review Checklist

ci:next
